### ISSUE TITLE
Phar file is broken

### ISSUE THREAD

**Issue by jderusse** [FriendsOfPHP/PHP-CS-Fixer#3004]
The official documentation recomend to use `http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar` but the URL does not returns the last version of cs-fixer.

```
$ curl -s http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar > /usr/local/bin/cs-fixer  && chmod +x /usr/local/bin/cs-fixer

$ cs-fixer
PHP CS Fixer 2.4.0 Silver Gingerbread by Fabien Potencier and Dariusz Ruminski (63661f3)
```

whereas CS fixer 2.5.0 is released 3 days ago

**Comment by SpacePossum** [comment]
Thanks for reporting!
I've confirmed this, as @keradus is currently the only who can resolve this I'm pinging him here.
It will probably resolved after the weekend earliest I'm afraid.

**Comment by jderusse** [comment]
It's not a big issue. Thank you for your work.

Is there a way to automatically update the website with the release process to avoid such issue in the next deployments?

**Comment by SpacePossum** [comment]
> Is there a way to automatically update the website with the release process to avoid such issue in the next deployments?

There is some [automation](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.5/.travis.yml#L121) for this in place, but it seems it failed us this time. The new minor version release showed some issues in our deployment scripts and update systems we didn't [anticipated ](https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2997).
Some of those we are trying to resolve through updating the code of the PHP CS Fixer itself, some need to be resolved in another matter as we've some limitations on the official sites we release on.
While sadly it takes some more time to get the latest version to our users, it should all be resolvable within the near future.

**Comment by keradus** [comment]
building was triggered: https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/267204092#L593
yet not executed properly.

website simply provide last build that works fully.
sadly, with last release we crashed `phar` file, thus it cannot build website to newest version.
We need to fix a phar first.

I changed title of this issue and assigned it to me.
when phar will be fixed, we will release newest version (and that would retrigger website fixing your original issue)
